docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -774,7 +775,7 @@
               register: new_reboot
               when: waiting_after_new_reboot.failed
 
-            when: waiting_after_reboot.failed
+            when: waiting_after_reboot.failed or waiting_after_new_reboot.unreachable
```

Comment:
Suggest to change the task name of line 764 to avoid two tasks with a same name - confuse to read the log

Reply:
Changes are done